src/libstd/sys/redox/net/mod.rs

```diff
@@ -20,10 +20,11 @@ use vec::{IntoIter, Vec};
 
 use self::dns::{Dns, DnsQuery};
 
-pub extern crate libc as netc;
```

Comment:
We no longer use the `libc` crate for `netc` definitions. They are now inside the `netc` module.